mod/consensus-types/pkg/types/block_test.go

```diff
@@ -43,6 +43,7 @@ func generateValidBeaconBlock() *types.BeaconBlock {
 		Body: &types.BeaconBlockBody{
 			ExecutionPayload: &types.ExecutionPayload{
 				Number:    10,
+				Timestamp: 10,
```

Comment:
nit: consider dropping Number if it's not relevant anymore. It makes the UT marginally clearer.